wiki/Digital_Transformations_in_Healthcare_Professional's_Work.qmd

```diff
@@ -0,0 +1,42 @@
+---
+title: "Digital Transfomations in Healthcare Professional's Work"
+subtitle: "BA New Media / Online Communication KF/BF A: Spring Term 2023"
+author: Laura Schomburg
+categories:
+  - "Media Psychology"
+  - "Digital Media"
+  - "Well Being"
+---
+
+
+Select a (K) Key Quote from the article that provides a good summary of the article. The Key Quote should be a quote of one sentence and not a whole paragraph.
+
+"In unexpected situations, control by professional norms effectively overruled control by the standards and guidelines encoded in digital objects."
+
+
+Describe in one paragraph what is the main (A) Argument of the article from which you have picked the Key Quote. Summarize in your own words the main theoretical argument expressed in this article. Here you do not use any direct citations.
+
+The article's main argument discusses the question how users of digital information systems (IS) in hospitals perceive the support of technology as advantageous or obstructive in their daily routines. The examined IS focus on features and use cases such as those that coordinate surgeries and manage the prioritization of patients based on their diseases to improve the internal process flows. The aforementioned benefits manifest in an improved operating model and smoother chains of activities provided by the additional knowledge within the IS. The article, in particular, sheds light on the visibility management approach which centers around the idea of the illustration of processes and workflows to increase the overall operational transparency. However, the majority of participants criticizes the resulting increase in transparency since it is accompanied by a perceived feeling of control which leads to a reduction of the individualâ€™s autonomy. The alleged loss of autonomy is not applicable in critical situations because, ultimately, the decision-making power is always held by the physicians during their surgical interventions. In summary, the usage of IS is evaluated ambiguously by the users in a hospital due to the materialized positive benefits in process flows on the one hand and the biased perception of an increased monitoring of the individual on the other hand.
+
+
+Discuss the (R) Relationship between the argument in the study and the theoretical arguments put forward in the other articles that you have read from this research area.
```

Comment:
Try to highlight two or three of the most important factors that are found to contribute to and two or three that harm individuals' autonomy need experiences.

Reply:
Contribution to Individual's autonomy:
-personal development contributes to the physicians' ability to better use their autonomy in hot situations. 
-Responsibility grows through visibility management as it leads to greater transparency in process flows

Harms Individual's autonomy:
-new technologies due to a higher level of controll
-new technologies that automatically make decisions about surgical procedures for physicians.
->affected by visibility management

Reply:
Huber, C., & Gärtner, C. (2018). Digital transformations in healthcare professionals’ work: Dynamics of autonomy, control and accountability. _Management Revue_, 29(2), 139-161.

Reply:
The new information systems (IS) make everyday decisions that used to be made by doctors, so doctors have been deprived of some of their autonomy (at least that's how they feel). Only in the case of really urgent decisions can they still put their autonomy above that of the IS.